upstream-community-operators/hpa-operator/hpa-operator.v0.1.6.clusterserviceversion.yaml

```diff
@@ -0,0 +1,154 @@
+    mediatype: image/png
+  installModes:
+  - type: OwnNamespace
```

Comment:
OwnNamespace should always be set to true

Reply:
fixed